crates/nomination/src/lib.rs

```diff
@@ -275,10 +292,25 @@ impl<T: Config> Module<T> {
         Ok(())
     }
 
+    pub fn request_nominator_withdrawal(
```

Comment:
Since we are no longer considering this two-phase withdrawal I think this can be renamed.